Dockerfile

```diff
@@ -1,17 +1,20 @@
 FROM golang:alpine
 
 # Create application directory
-RUN mkdir /app
-ADD . /app/
-WORKDIR /app
+RUN mkdir /src
+ADD . /src/
+WORKDIR /src
 
 # Build the application
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go mod download
```

Comment:
Could you verify that building the image works, even with an empty cache, if this line is removed? I think this is required when using Go version >= 1.16. If it works with an empty cache as well, we are good to go! 🙂

Reply:
Alright, I deleted the old image then ran `docker builder prune`, then run `docker build --no-cache -t prestrafe-gsi:dev .`

I also modified Dockerfile's `FROM golang:alpine` to `FROM golang:1.16-alpine` to make sure that it runs on the 1.16 version and it still works. 

I believe that `go mod download` (or something similar) is already included while running `go build`, though `go build` does this faster by itself, and the reason for that command to exist in the first place is to improve caching between rebuilds... but so far it seems to be doing the opposite, weird.